### PR TITLE
Bump to v4.4.0

### PR DESCRIPTION
**Pull request by ashley-cui**
Final release notes and bump to v4.4.0

```release-note
None
```

### CODE REVIEW

**Comment by ashley-cui** [comment]
@containers/podman-maintainers PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -93,52 +96,59 @@ The `podman kube down` command accepts a new option, `--force`, which removes vo
...
 - Fixed a bug where named volumes were not properly idmapped.
 - Fixed a bug in `podman kube play` where the sdnotify proxy could cause Podman to deadlock ([#16076](https://github.com/containers/podman/issues/16076)).
 - Fixed a bug where the containers.conf files are reloaded redundantly.
 - Fixed a bug where `podman system df` reported wrong image sizes ([#16135](https://github.com/containers/podman/issues/16135)).
 - Fixed a bug where `podman inspect` did not correctly remote the IPCMode of containers ([#17189](https://github.com/containers/podman/issues/17189)).
 - Fixed a bug where containers created in a pod using the `--userns keep-id` option were not correctly adding username entries to /etc/passwd within container ([#17148](https://github.com/containers/podman/issues/17148)).
+- Fixed a bug where the `--publish-all` flag in the `podman create` and `podman run` commands would occasionally assign colliding ports.
+- Fixed a bug where `podman machine init –image-path` on Windows was not correctly handling absolute paths ([#15995](https://github.com/containers/podman/issues/15995)).
```
Comment (Luap99) [inline RELEASE_NOTES.md:124]:
```suggestion
- Fixed a bug where `podman machine init --image-path` on Windows was not correctly handling absolute paths ([#15995](https://github.com/containers/podman/issues/15995)).
```

Comment (ashley-cui) [reply RELEASE_NOTES.md]: Fixed, good catch!

**Review by Luap99 — COMMENTED** [review]
LGTM

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -3,20 +3,19 @@
...
 ### Features
 - Introduce Quadlet, a new systemd-generator that easily writes and maintains systemd services using Podman.
-- The `podman kube play` command now supports hostPID in the pod.spec ([#17157](https://github.com/containers/podman/issues/#17157)).
+- The `podman kube play` command now supports hostPID in the pod.spec ([#17157](https://github.com/containers/podman/issues/17157)).
 - The `podman build` command now supports the `--group-add` option.
 - A new command, `podman network update` has been added, which updates networks for containers and pods.
-- The `podman network create` command now supports a new option, `--network-dns-server` sets the DNS servers that this network will use.
-- The `podman kube play` command now accepts the`--publish` option, in order to set or override port publishing.
+- The `podman network create` command now supports a new option, `--network-dns-server`, which sets the DNS servers that this network will use.
+- The `podman kube play` command now accepts the`--publish` option, which to sets or overrides port publishing.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:10]:
```suggestion
- The `podman kube play` command now accepts the`--publish` option, which sets or overrides port publishing.
```

**Comment by mheon** [comment]
/lgtm
/hold

Restarted two flakes

**Comment by ashley-cui** [comment]
Oof, re-pushed right as the other comment landed. Oh well.

**Comment by ashley-cui** [comment]
Re-started another flake, @mheon could you give this another slash lgtm if it lgty, thanks!

**Pull request merged**